x/evm/keeper/querier.go

```diff
@@ -197,12 +197,8 @@ func queryExportAccount(ctx sdk.Context, path []string, keeper Keeper) ([]byte,
 		return nil, err
 	}
 
-	balanceInt := keeper.GetBalance(ctx, addr)
-	balance := sdk.NewIntFromBigInt(balanceInt)
-
 	res := types.GenesisAccount{
 		Address: hexAddress,
-		Balance: balance,
 		Code:    keeper.GetCode(ctx, addr),
 		Storage: storage,
```

Comment:
is there a reason why we don't return the balance on export account? (not saying its wrong but wanted to know why)

Reply:
I removed it because it's redundant with the account balance. Removing it reduces the size of the genesis json. Also because of the invariant, it ensures that the accounting is consistent. I added a note on the struct about that